src/components/application_manager/include/application_manager/hmi_capabilities_impl.h

```diff
@@ -238,10 +238,10 @@ class HMICapabilitiesImpl : public HMICapabilities {
 
   bool DeleteCachedCapabilitiesFile() const OVERRIDE;
 
-  std::set<hmi_apis::FunctionID::eType> GetDefaultInitializedCapabilities()
+  std::set<hmi_apis::FunctionID::eType> GetCapabilitiesToRequest()
```

Comment:
@LitvinenkoIra Why this function is named `GetCapabiltites` ? It returns a set of function IDs but not the capabilities.